e2e/sabactl_test.go

```diff
@@ -19,21 +19,15 @@ import (
 
 const (
 	// ExitSuccess represents no error.
-	ExitSuccess subcommands.ExitStatus = subcommands.ExitSuccess
```

Comment:
Please get rid of `github.com/google/subcommands` as it was replaced with cobra.